test-packages/test-services/openapi/swagger-yaml-service/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "swagger-yaml-service",
+  "version": "1.37.0",
```

Comment:
This version and the sdk version in line 26 rely on the current version, which is dynamic.
Every release will push changes like [this](https://github.com/SAP/cloud-sdk-js/commit/f9ad1fc1323f84c9b261e4cbf8bc2953c9a8f71d) automatically via this [script](https://github.com/SAP/cloud-sdk-js/blob/main/scripts/after-bump.js)
Without changing the script, the build of this commit will fail.
So we either adjust the script to update this json file or provide more parameters to the generator to generate static versions.
I would vote for static versions if the package json is just used for the e2e tests.

Reply:
yes good point. I will add one more flag to the generator to pin the version.

Reply:
ah there is already the `  versionInPackageJson` flag which I can use.

Reply:
and for the test we have at the moment the package.json can go way completely.